### PR TITLE
MAISTRA-706: remove jaeger hub/tag values from values.yaml and parame…

### PR DESCRIPTION
**Pull request by knrc**
…terise image names

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: tmp/build/patch-jaeger.sh [hunk 1]
```diff
@@ -4,7 +4,10 @@ function jaeger_patch_values() {
 	# update jaeger zipkin port name
   sed -i -e '/service:$/,/externalPort:/ {
     s/name:.*$/name: zipkin/
-	}' ${HELM_DIR}/istio/charts/tracing/values.yaml
+	}' -e '/jaeger:$/,/^[^ 	]/ {
```
Comment (dgn) [inline tmp/build/patch-jaeger.sh:7]: oh, I missed this one, thank you!

**Review by dgn — APPROVED** [review]

**Review by objectiser — APPROVED** [review]

File: helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml [hunk 2]
```diff
@@ -12,8 +12,8 @@ spec:
   strategy: production
 
   query:
-    {{- if and .Values.jaeger.hub .Values.jaeger.tag }}
-    image: {{ .Values.jaeger.hub }}/jaeger-query:{{ .Values.jaeger.tag }}
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag .Values.jaeger.jaegerQueryImage }}
```
Comment (objectiser) [inline helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml]: nit: `.Values.jaeger.jaegerQueryImage` -> `.Values.jaeger.queryImage`?
and similar for the other image names below.

Comment (knrc) [reply helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml]: Should be renamed now

**Comment by knrc** [comment]
Squashed commits to merge

**Pull request merged**